Gems/GradientSignal/Code/Source/Editor/EditorImageGradientComponent.cpp

```diff
@@ -372,7 +373,7 @@ namespace GradientSignal
 
     void EditorImageGradientComponent::EnableComponentMode()
     {
-        if (m_componentModeDelegate.AddedToComponentMode())
+        if (m_componentModeDelegate.IsConnected())
```

Comment:
Excellent you can use this new API now, hopefully it's doing what you need 👍

Reply:
Yep it's perfect, thanks!

Reply:
Awesome to hear 😄  Great work @amzn-ahmadkrm